python/simulation.py

```diff
@@ -1316,6 +1316,24 @@ def _add_near2far(self, fcen, df, nfreq, near2fars):
     def get_farfield(self, f, v):
         return mp._get_farfield(f.swigobj, py_v3_to_vec(self.dimensions, v, is_cylindrical=self.is_cylindrical))
 
+    def get_farfields(self, near2far, resolution, where=None, center=None, size=None):
+        vol = self._volume_from_kwargs(where, center, size)
+        result = mp._get_farfields_array(near2far.swigobj, vol, resolution)
+        res_ex = result[0] + result[1] * 1j
```

Comment:
Since we do this a lot, to avoid allocating additional temporary arrays, we might want a function like:
```py
def complexarray(re, im):
    z = im*1j
    z += re # add re to z in-place
    return z
```

Reply:
Done.